Make the installation script idempotent

volAV opened the pull request.

mshkrebtan commented on install.sh:
```diff
@@ -4,48 +4,69 @@
 }
 
 # Bash
-ln -s ${DIR}/bash/.bashrc ${HOME}
```
@valAV Why do you remove this command?

`.bashrc` is executed when an interactive non-login shell starts.

volAV replied:
macOS terminal by default runs a login-shell and _.bashrc_ will never be used.

mshkrebtan reviewed (commented):
We have to clarify the need of Bash startup files.

volAV committed:
Make install script idempotent

- Rewrite installation script to have the idempotent behavior
- Rename all resources which names starting with dot

mshkrebtan commented:
Closing this as I am switching to `make` for installation.
